### ISSUE TITLE
End dimension cannot be chunkloaded

### ISSUE THREAD

**Issue by aphroidite** [ChosenArchitect/Project-Architect-2#141]
**Describe the bug**

->  Every other dimension than the end can be chunkloaded (so you can set up quarries/farms).
I've claimed a small a mount of chunks for an enderman farm (one of the fastest ways to get exp in the pack, layering enderman spawn platforms with vector plates) and force loaded the zone to get the drops/exp back in the overworld through ender chests/quantum entangloporter/me quantum ring/entangled block. The moment a player is not present there the dimension stops completely no matter the settings.
Tested both before/after killing the dragon and the chaotic one.
And it's really, if not the best, one of the best ways to get exp, I got a Superior black hole tank fully filled in a couple minutes...

**To Reproduce**

1. Claim chunks in the nether and force-load them.
2. Make a spawn platform for enderman/stack of platforms.
3. Set up any way to transfer items/fluids across dimensions (Even EMC links)
4. Leave the dimension.

**Expected behavior**

-> As I set up the pipes/laser nodes, ender pearls should be piped into a matter condenser and then and EMC link, dimlets should be directly transfered to my ME system (several import buses on ender chest) and liquid exp should be piped into my overworld tanks yet none of them get transfered once the dimension has no players.

**Modpack version**
-> 3.2
**Java Version**
-> 8 build 361
**Modpack Launcher Used**
-> Curseforge
**Ram Allocated**
-> 10gb

**Comment by ChosenArchitect** [comment]
I did test chunk loading in the end it is loading

**Comment by M4trixm4n** [comment]
If i'm not wrong, this isn't an issue about chunkloading. In general, mobs will not spawn naturally without a player nearby. That's not the case for mob spawners though.

**Comment by aphroidite** [comment]
I tested using a quarry there tho, it stops as well.

**Comment by IRicTheCoder** [comment]
Didn't get any more feedback on this, I have made a lot of tests with chunk loading and there is a lot of things that can cause this issue. The Quest Book now explains how to change settings for fake players to work, and there is information for EMC Links not working while offline.

As for other issues, most of the time something is missing. Closing this as solved, reopen if the issue persists.

Edit: Note that as of 4.0 this issues seem to be a lot less frequent.
